day2/test/test_gift.py

```diff
@@ -0,0 +1,22 @@
+from unittest import TestCase, main
+from day2.gift import Gift
+
+
+class Test_gift(TestCase):
+    def setUp(self):
+        self.gift = Gift()
+
+    def test_one_gift(self):
+        self.assertEqual(
+            self.gift.calculation_of_wrapping_paper(['2x3x4']), 58)
+
+    def test_two_gift(self):
+        self.assertEqual(
+            self.gift.calculation_of_wrapping_paper(['2x3x4', '1x1x10']), 101)
+
+    def test_value_with_line_break(self):
+        self.assertEqual(
+            self.gift.calculation_of_wrapping_paper(['2x3x4\n']), 58)
```

Comment:
Pelo que entendi do motivo que vc falou para usar aquele `for` lá em cima, esse comando 
```python
open("input.txt").readlines()
``` 
retorna um array tipo assim:
```python
['2x3x4\n', '5x8x12\n', '15x10x3\n']
```
Certo?